test/libsolidity/semanticTests/smoke/fallback.sol

```diff
@@ -9,18 +9,18 @@ contract A {
     }
 }
 // ====
-// compileViaYul: also
 // compileToEwasm: also
+// compileViaYul: also
 // ----
 // data() -> 0
 // ()
 // data() -> 1
 // (): hex"42ef"
 // data() -> 2
 // externalData() -> 0x20, 2, left(0x42ef)
-// balance() -> 0
```

Comment:
Should remove the balance function in this contract.

Reply:
I'll revert.

Reply:
Ah, it seems you cannot call a function called `balance` anymore.